acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -2,6 +2,18 @@
     visitHostsPage()
 end
 
+When(/^I fill in the Host Name field with the default host name$/) do
+    fillInDefaultHostAndPort()
```

Comment:
Let's refactor this a bit to apply the [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) principle w.r.t. the page elements, because that will make maintenance a little easier if we ever rearrange the add/edit host dialog.

So instead of `fillInDefaultHostAndPort()` call a more general function and pass it the value you want to use for default - something like `fillInHostAndPort("172.17.42.1:4979")`, that line 18 use  `fillInHostAndPort(hostName)`. The point is to encapsulate the call to `@hosts_page.hostName_input.set` in a single method.